File: src/utils/fileUtils.js
const fs = require('fs');

function readFile(filePath) {
  return JSON.parse(fs.readFileSync(filePath, 'utf8'));
}

function writeFile(filePath, content) {
  fs.writeFileSync(filePath, content);
}

function fileExists(filePath) {
  return fs.existsSync(filePath);
}

module.exports = { readFile, writeFile, fileExists };

File: src/utils/fileUtils.py
import json
import os

def read_file(file_path):
    with open(file_path, 'r') as file:
        return json.load(file)

def write_file(file_path, content):
    with open(file_path, 'w') as file:
        json.dump(content, file)

def file_exists(file_path):
    return os.path.exists(file_path)
